public/components/stub-modal/stub-modal.html

```diff
@@ -114,25 +114,28 @@ <h4 class="modal-title">{{ modalTitle }}</h4>
     </div>
 
     <div class="modal-footer" ng-if="mode !== 'import'" ng-hide="actionInProgress || (actionSuccess && composerUrl)">
+        <button type="submit" class="btn btn-default" id="testing-do-generic-submit" ng-click="submit(stubForm)" ng-show="false">
+            Submit
+        </button>
         <div class="pull-left">
             <button type="button" class="btn btn-default pull-left" data-dismiss="modal" ng-click="delete()" ng-if="stub.id">Delete</button>
             <button type="button" class="btn btn-default" data-dismiss="modal" ng-click="cancel()">Cancel</button>
         </div>
         <div class="pull-right">
-            <button type="submit" class="btn btn-default" id="testing-create-stub" ng-click="ok(addToComposer=false,addToAtomEditor=false)" ng-disabled="disabled || stubForm.$invalid " ng-show="stub.status === 'Stub'">
+            <button type="button" class="btn btn-default" id="testing-create-stub" ng-click="submit(stubForm)" ng-disabled="disabled || stubForm.$invalid " ng-show="stub.status === 'Stub'">
```

Comment:
Minor: if only one of these buttons is displayed at the same time, could we remove the click handlers, make them type `submit`, and bind the submit function to the form with `ng-submit={submit}` once?

Reply:
(looking at the code below, there's a chance this might not play nicely with `actionSuccess`)